### ISSUE TITLE
Where is "track_running_stats" implementation code?

### ISSUE THREAD

**Issue by junha1125** [vacancy/Synchronized-BatchNorm-PyTorch#47]
Hi! 

I looked for "track_running_stats" implementation code. But I could not find.
Since SynchBN inherit nn.modules.batchNrom, It automatically track running statics?

Thank you for your nice code.

**Comment by vacancy** [comment]
https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/blob/master/sync_batchnorm/batchnorm.py#L68-L70

Yes. It always track the running stats.
